x-pack/plugins/ingest_manager/server/services/package_config.test.ts

```diff
@@ -161,4 +164,32 @@ describe('Package config service', () => {
       ]);
     });
   });
+
+  describe('update', () => {
```

Comment:
Thanks! I tried to add this the other day and couldn't work it out. Thanks for the test and example.

cc @ericdavisx

Reply:
Of course!